site/instrument.ts

```diff
@@ -45,4 +47,16 @@ if (LOAD_SENTRY) {
         replaysSessionSampleRate: sampleRate,
         replaysOnErrorSampleRate: 0,
     })
+    updateSentryTags()
+    updateSentryUser()
+}
+
+function updateSentryTags() {
+    updateSentryReferrerTag()
+    updateSentryExperimentTags()
+}
+
+function updateSentryReferrerTag() {
+    const ref = document.referrer ? new URL(document.referrer).hostname : "none"
+    Sentry.setTag("referrer", ref)
```

Comment:
Rather than using a custom value for the absence of a referrer, it's IMO better to not set it at all. Sentry allows filtering for absence of a tag with `!has:my_tag`.